### PR TITLE
DM-45443: First draft

### PR DESCRIPTION
**Pull request by timj**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,9 +1,80 @@
...
+
+The role of the Build Engineering Team is to facilitate releases of the LSST Science Pipelines software to the community and other parts of the Rubin organization.
+This includes the support and evolutions of:
+
+* Making formal science pipelines releases (working with the pipelines team on release notes and timing as well as the infrastructure for making the releases).
+* The Conda python environment (e.g., the `rubin-env` meta package)
+* The pipeline build system used by the developers (`sconsUtils`, EUPS, lint tooling, `lsstsw`, `lsstinstall`)
+* The GitHub Actions and associated template packages for pipelines packages.
+* The Python/C++ bindings such as Pybind11 or Nanobind.
+* The CI/CD infrastructure (nightly, weekly, ticket branch releases, making docker containers, running AP verification jobs) on multiple architectures.
```
Comment (ktlim) [inline index.md]: I think [the shared stack script](https://github.com/lsst-dm/shared-stack/blob/main/shared_stack.bash) may have to be here, although responsibility for deploying it and doing updates to shared stacks could be handled by consumers (Summit and Base HTCondor, USDF).

File: index.md [hunk 2]
```diff
@@ -1,9 +1,80 @@
...
+A detailed overview of the build system can be found in {cite:p}`DMTN-218`.
+
+The BET can investigate new technologies for building or deploying the software such as:
+
+* Changing from Docker containers to a new technology as driven by stakeholders.
+* Using a completely different CI/CD system.
+* Having alternate distributions mechanisms that do not use EUPS (such as Conda).
+* Replacing SCons as the package build system.
+
+As we head into the era of long-lived formal data releases there is an expectation that BET would be involved in any changes to how we coordinate branches and release strategies in Git.
```
Comment (ktlim) [inline index.md]: Changes in organization of packages are a DM-CCB function but would also be heavily informed by the BET.

File: index.md [hunk 3]
```diff
@@ -1,9 +1,80 @@
...
+* Supporting of the "shared stack" at the USDF by the pipeline developers.
+
+## Requirements
+
+There are no Service Level Agreements between BET and other parts of the Rubin organization.
+Furthermore, there are no formal requirements for build metrics or deployment mechanisms written in {cite:p}`LSE-61`.
+
+Informal assumed requirements are:
+
+* Developers shall have a CI system to check their code before they merge it.
```
Comment (ktlim) [inline index.md:138]: The code, build (if necessary), install (if necessary), test, debug cycle should be as rapid as possible to minimize developer overhead.

File: index.md [hunk 4]
```diff
@@ -1,9 +1,80 @@
...
+The Telescope and Site software team use the Science Pipeline container as the core of the CSC containers that they deploy to the summit.
+Any changes to Python major versions must be approved and coordinated by T&S as well as with the Science Pipelines team.
+
+The Rubin Science Platform deploys the pipeline docker containers.
+These containers form the primary interface used by the science community.
+The RSP community, including the Community Science Team, expects to have access to containers from weekly releases, formal validated releases, and recent dailies.
+
+Finally, science users who are not using the containers need to be able to install the software.
+The requirement is that the software be installable to outside users but this system can change over time.
+
```
Comment (ktlim) [inline index.md:122]: Might list DM-CCB here as a stakeholder in the sense of relying on the release process as well as relying on BET expertise to inform decisions.

File: index.md (hunk 3 above)
Comment (ktlim) [reply index.md:138]: This is more than just CI, it also goes to how eups and sconsUtils work (together).

Comment (timj) [reply index.md:138]: The requirement is CI. The requirement isn't that eups and sconsUtils exist.

Comment (ktlim) [reply index.md:138]: There's no requirement for eups and sconsUtils, of course.  But being able to build and test locally, without doing a remote build somewhere, seems critical to cycling rapidly.

File: index.md [hunk 5]
```diff
@@ -1,9 +1,149 @@
...
+The BET supports and evolves the core technologies and can propose alternatives such as nanobind.
+
+### Stack Builds
+
+Since the LSST Science Pipelines consist of separate git repositories with defined dependencies, software tooling is required to build a complete release and ensure that the packages are built in the correct order.
+BET currently enables this with the `lsstsw` tool.
+
+Additionally, once package source and binary releases have been made the `lsstinstall` tool is provided to allow external users to install the software.
+This also requires the use of `eups distrib install`.
+
```
Comment (roceb) [inline index.md:75]: Should we add a line about maintaining https://eups.lsst.codes/

**Pull request merged**